src/Mod/Arch/Resources/ui/preferences-ifc.ui

```diff
@@ -466,7 +491,8 @@
       <item>
        <widget class="Gui::PrefCheckBox" name="checkBox_13">
         <property name="toolTip">
-         <string>If this is checked, all FreeCAD object properties will be stored inside the exported objects, allowing to recreate a full parametric model on reimport.</string>
+         <string>All FreeCAD object properties will be stored inside the exported objects,
+allowing to recreate a full parametric model on reimport. </string>
```

Comment:
Remove trailing whitespace

Reply:
Note addressed in 363f985

Reply:
I am still a newbie in FC. Why are these trailing whitespaces important?
I enabled modifications by developers so if you like you can immediately commit such fixes.

GitHub tells me now also
"New! Suggest specific code changes that the pull request author or assignees can immediately commit. You will be attributed in the commit."

So if you would use this I could quickly commit it.